ui/src/notebooks/pipes/Query/index.ts

```diff
@@ -13,7 +13,7 @@ register({
     activeQuery: 0,
     queries: [
       {
-        text: '',
+        text: '// Write Flux script here',
```

Comment:
do we want to write this for all of them, or just the first one?

Reply:
I figure all, cuz it doesn't hurt